### PR TITLE
check god role exist when meta init

### PR DESCRIPTION
**Pull request by nevermore3**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Issue(s) number: 
close https://github.com/vesoft-inc/nebula-ent/issues/957

#### Description:
Before:
 it will check whether there is a root account，when meta is restarted

 After:
 it will check whether there is an account with the GOD role， when the meta is restarted

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/meta/RootUserMan.h [hunk 1]
```diff
@@ -20,6 +20,27 @@ namespace meta {
  * */
 class RootUserMan {
  public:
+  static bool isGodExists(kvstore::KVStore* kv) {
+    auto rolePrefix = MetaKeyUtils::roleSpacePrefix(kDefaultSpaceId);
+    std::unique_ptr<kvstore::KVIterator> iter;
+    auto code = kv->prefix(kDefaultSpaceId, kDefaultPartId, rolePrefix, &iter, false);
+    if (code != nebula::cpp2::ErrorCode::SUCCEEDED) {
+      LOG(INFO) << "Prefix God Role Failed";
+      return false;
```
Comment (SuperYoko) [inline src/meta/RootUserMan.h:29]: Will god user be created again if this read failed?  Check plz.

**Review by SuperYoko — CHANGES_REQUESTED** [review]
May break user info if kv read failed temporary for this check.

File: src/meta/RootUserMan.h (hunk 1 above)
Comment (nevermore3) [reply src/meta/RootUserMan.h]: fix

**Review by SuperYoko — APPROVED** [review]

**Review by critical27 — APPROVED** [review]

**Comment by critical27** [comment]
Good job~

**Pull request merged**